README.md

```diff
@@ -477,17 +477,34 @@ Generally we release `MINOR` versions of Airflow from a branch that is named aft
 `2.7.*` releases are released from `v2-7-stable` branch, `2.8.*` releases are released from `v2-8-stable`
 branch, etc.
 
-Most of the time in our release cycle, when the branch for next `MINOR` branch is not yet created, all
+1. Most of the time in our release cycle, when the branch for next `MINOR` branch is not yet created, all
 PRs merged to `main` (unless they get reverted), will find their way to the next `MINOR` release. For example
 if the last release is `2.7.3` and `v2-8-stable` branch is not created yet, the next `MINOR` release
-is `2.8.0` and all PRs merged to main will be released in `2.8.0`. There is a brief period of time when we
-cut a new `MINOR` release branch and prepare the alpha, beta, RC candidates for the `2.NEXT_MINOR.0` version
-where PRs merged to main will only be released in the following `MINOR` release.
-
-However, some PRs (bug-fixes and doc-only changes) when merged, can be cherry-picked to current `MINOR` branch
-and released in the next `PATCHLEVEL` release - for example when the last released version from `v2-7-stable`
-branch is `2.7.2`. Some of the PRs from main can be marked as `2.7.3` milestone by committers, the release manager
-will try to cherry-pick them into the release branch. If successful, they will be released in `2.7.3`.
+is `2.8.0` and all PRs merged to main will be released in `2.8.0`. However, some PRs (bug-fixes and
+doc-only changes) when merged, can be cherry-picked to current `MINOR` branch and released in the
+next `PATCHLEVEL` release for example `2.8.1` is already released and we are working on `2.9.0dev`,
+marking a PR with `2.8.2` milestone will mean that it will be cherry-picked to `v2-8-test` branch and
+released in `2.8.2rc1` and eventually in `2.8.2`.
+
```

Comment:
```suggestion
next `PATCHLEVEL` release. For example, if `2.8.1` is already released and we are working on `2.9.0dev`,  then
marking a PR with `2.8.2` milestone means that it will be cherry-picked to `v2-8-test` branch and
released in `2.8.2rc1`, and eventually in `2.8.2`.
```